.env.dev.template

```
@@ -20,9 +20,9 @@ DB_PORT_ASSESSMENT=5435
 # Certificate
 DB_HOST_CERTIFICATE=localhost
 DB_PORT_CERTIFICATE=5439
-# Template
-DB_HOST_TEMPLATE_SERVER=localhost
-DB_PORT_TEMPLATE_SERVER=5437
+# Example
+DB_HOST_EXAMPLE_SERVER=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_HOST_EXAMPLE_SERVER key should go before the DB_HOST_SELF_TEAM_ALLOCATION key

(UnorderedKey)